### DISCUSSION TITLE
How do we add a cross-reference to an embedded video?

### DISCUSSION THREAD

**Discussion by bryanhanson** [quarto-dev/quarto-cli#6208] [Q&A]
### Description

I don't see this discussed anywhere, apologies if I've missed it.  If I embed a video via:

```
{{< video my-video.mp4 >}}
```

how can I cross-reference it?

**Comment by mcanouil** [comment]
Currently, you can't but it should be possible in the final 1.4 version.

**Reply by cscheid** [reply]
Sneak peek (edit: from an unreleased branch I'm actively working on, not currently available for public consumption)!

<img width="852" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/15e2ed38-7c83-46cf-a19b-8917a22f61f4">

**Comment by bryanhanson** [comment]
Great!  Is this in the dev version?  I always use the dev version, but I guess I don't know how to access the docs for dev.

**Reply by mcanouil** [reply]
It's not released, you'll have to wait.

**Reply by cscheid** [reply]
If you're interested, the issue to follow is https://github.com/quarto-dev/quarto-cli/issues/4944.
